geode-redis/README.md

```diff
@@ -25,7 +25,7 @@ and then connect the [Redis-CLI](https://redis.io/topics/quickstart) to that ins
 
 ### <a name="building-apache-geode"></a>Supported Redis Commands
 
-Not all Redis commands are currently supported. The current set of supported Redis commands is listed [here](#redis-command-status).
+Not all Redis commands are supported. The current set of supported Redis commands is listed [here](#supported-redis-commands).
```

Comment:
An occurrence of 'current' remains. Suggested re-wording:
Not all Redis commands are supported. See [Supported Redis Commands](#supported-redis-commands).